### DISCUSSION TITLE
translation site web with .resx and .po dont work

### DISCUSSION THREAD

**Discussion by ODA669** [OrchardCMS/OrchardCore#13535] [Q&A]
I created my multilanguage website, activate the localization and now I need to know how I can do so that with a button it activates the language change from English to French and in turn recognizes the .resx files so that it generates specific translations, the IStringLocalizer can find the resourses

**Comment by hishamco** [comment]
Try to register the `ResourceManagerStringLocalizer` into DI

**Reply by jtkech** [reply]
By defaut we don't support `*.resx` files at the tenant/modules level, only `*.po` files, so it may work in debug/dev mode if a module registers later on the resource localization services, but it will not work in release/prod mode because `.resx` files will not be embedded in the module assembly.

https://github.com/OrchardCMS/OrchardCore/blob/4b4f95e423eaf3adf5d211c0e1f5a231f30856ff/src/OrchardCore/OrchardCore.Module.Targets/OrchardCore.Module.Targets.targets#L11

**Reply by hishamco** [reply]
Maybe I didn't notice that before, all I remembered is we disabled the `ResourceManagerStringLocalizer` by default not the `resx`, but it's good tip for me first and others of sure

**Reply by ns8482e** [reply]
You can always manually change to embed .resx at project level

**Reply by ODA669** [reply]
> You can always manually change to embed .resx at project level

i try , my project has 3 module orchardcore.cms.web, one module  that i created and one theme tha i created, i try to read the .resx with @inject IStringLocalizer<MyController> localizer, but dont work, i try to Ngettext  @using NGettext.Loaders
@using NGettext.Plural , dont work and i dont know why the project dont find any folder resources and localization and its include in the project, but the cultures works fine, ièm new in Ochardcore platform, can you say me how i fix that ?

**Reply by hishamco** [reply]
`Ngettext`?!!

**Comment by ODA669** [comment]
i try with the .po but the module don't read the files, i try to insert in orchardCore.cms.web/localization but the file its exclude to project i don't know how  when i try to include don't work,  i created file localization in new module and added the .po files  and the iStringLocalizer dont find the file, i new in this platform, do you can explain me or show me how  i can fix that , thanks for answer.
